vetur.config.js

```diff
@@ -1,7 +1,9 @@
 // vetur.config.js
 /** @type {import('vls').VeturConfig} */
 module.exports = {
+  
   projects: [
+    './src',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8560396)
```suggestion
    "./src",
```